version.props

```diff
@@ -1,6 +1,6 @@
 ﻿<Project>
   <PropertyGroup>
-    <VersionPrefix>2.1.2</VersionPrefix>
+    <VersionPrefix>2.1.1</VersionPrefix>
```

Comment:
Why is this 2.1.1?

Reply:
Per policy change in https://github.com/aspnet/Home/issues/3316, IISIntegration will not need to ship new packages in 2.1.3...unless I'm mistaken. Can you confirm you're not expecting IISIntegration patches to land in 2.1.3?

Reply:
No patches in 2.1.3. And glad we are making this change, though the jagged edges just move to a new place 😢 .